src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM_SAIL = 20;
+  const LONG_TERM_SAIL = 50;
```

Comment:
Potentially incorrect variable naming. Variable 'LONG_TERM_SAIL' should be 'LONG_TERM_SALE'.